features/admin.organizations.v1/pages/organizations.tsx

```diff
@@ -111,6 +127,8 @@ const OrganizationsPage: FunctionComponent<OrganizationsPageInterface> = (
     const [ authorizedListPrevCursor, setAuthorizedListPrevCursor ] = useState<string>("");
     const [ authorizedListNextCursor, setAuthorizedListNextCursor ] = useState<string>("");
     const [ activePage, setActivePage ] = useState<number>(1);
+    const [ shouldShowMetaAttributeComponent, setShouldShowMetaAttributeComponent ] = useState<boolean>(false);
```

Comment:
Let's also define a feature flag to conditionally enable/disable meta attribute component.

Reply:
This is now updated. A **isFilterByMetadataAttributesEnabled** feature flag is added.